framework/src/outputs/FileOutput.C

```diff
@@ -148,12 +148,11 @@ FileOutput::setFileBaseInternal(const std::string & file_base)
 
   // Check the file directory of file_base and create if needed
   // Check if _file_base is an absolute path
-  std::string base;
-  if (_file_base.find_first_of("/", 0) == 0)
-    base = "./" + MooseUtils::relativepath(_file_base);
+  std::string base = _file_base.substr(0, _file_base.find_last_of('/'));
```

Comment:
is there not something we can do with `std::filesystem` that is smarter and easier to follow?

Reply:
moved to `std::filesystem`